packages/devextreme/js/__internal/grids/grid_core/editing/m_editing.ts

```diff
@@ -96,7 +97,14 @@ class EditingControllerImpl extends modules.ViewController {
 
   _changes: any;
 
-  _deferreds: any;
+  /**
+   * A collection of deferred objects.
+   *
+   * Use with `waitForDeferredOperations` to await all deferred objects in this collection.
+   *
+   * Use `addDeferred` to add a deferred object to this collection.
+   */
+  _deferreds!: DeferredObj<any>[];
 
```

Comment:
Better not to duplicate info that we can get out of var names, types etc
But info about the purpose of this var would be useful

```suggestion
  /**
   * A collection of deferred objects that are awaited before …
   *
   * @see {@link waitForDeferredOperations}
   * @see {@link addDeferred}
   */
  private _deferreds!: DeferredObj<void>[];
```